cypress/integration/ui/login.spec.js

```diff
@@ -2,16 +2,16 @@
 
 describe('Login', () => {
   beforeEach(() => {
-    cy.visit('/')
-  })
-
+    cy.visit('/');
+  });
+//do not use cy.login() here
```

Comment:
formatting please? (Use space after `//` and align with the actual code, indent of 2, then 0, then 4 doesn't make sense.)

Reply:
Fixed :)